doc/man3/OPENSSL_malloc.pod

```diff
@@ -149,8 +152,16 @@ any effect, is must be called before any of the allocation functions
 (e.g., CRYPTO_malloc()) are called, and is therefore normally one of the
 first lines of main() in an application.
 
-If the library is built with the C<crypto-mdebug> option, then two additional
-environment variables can be used for testing failure handling.  The variable
+If the library is built with the C<crypto-mdebug> option, then one
+function and two two additional environment variables are available.
+for checking statistics and testing failure handling.
+
+The function CRYPTO_get_alloc_counts(), fills in the number of times
+each of CRYPTO_malloc(), CRYPTO_realloc(), and CRYPTO_free() have been
+called, into the value pointed to by B<mcount>, B<rcount>, and B<fcount>,
+respectively.  If any value is NULL, then the count is not stored.
```

Comment:
Is it actually right spot for inserting this? I mean it's inserted between strongly related sentences, one that says that there are environment variables and one that describes just mentioned variables...

Reply:
I'd swap value for pointer here.  The pointer is NULL, the value is undefined.

Reply:
I'm not sure. @dot-asm has a point but it isn't unreadable as it stands.

How about splitting so that the added function is mentioned and described first and then the addition of the two environment variable after?

 > If the library is built with the C<crypto-mdebug> option, an addition function is defined...


> If the library is built with the C<crypto-mdebug> option, two additional environment variable modify the operation...